### PR TITLE
HMConfig_AskSinPPCustom: support HB-UNI-Sensor-THPD-BME280

### PR DESCRIPTION
**Pull request by DarkShortie**

### CODE REVIEW

File: examples/custom/contrib/FHEM/HMConfig_AskSinPPCustom.pm [hunk 1]
```diff
@@ -72,6 +72,15 @@ $HMConfig::culHmRegDefine{"longActNum"}                   = {a=>0xA5,s=>1.0,l=>3
 $HMConfig::culHmRegDefine{"longActIntens"}                = {a=>0xAB,s=>1.0,l=>3,min=>0,max=>255,c=>'',p=>'y',f=>'',u=>'',d=>1,t=>"Volume or Led Brightness"};
 # Volume0=0, Volume10=20, Volume20=40, Volume40=60, .... Volume100=200
 
+# HB-UNI-Sensor-THPD-BME280
+$HMConfig::culHmRegDefine{"deviceLedMode"} = {a=>5.6,s=>0.2,l=>0,min=>0,max=>1,c=>'',p=>'n',f=>'',u=>'',d=>1,t=>"LED mode"};
+$HMConfig::culHmRegDefine{"lowBatLimit"} = {a=>18,s=>1,l=>0,min=>0.9,max=>5.0,c=>'',p=>'n',f=>10,u=>'V',d=>1,t=>"low battery limit"};
```
Comment (pa-pa) [inline examples/custom/contrib/FHEM/HMConfig_AskSinPPCustom.pm:77]: already exists https://github.com/mhop/fhem-mirror/blob/41fe9bbf317f54bccfc397e26224c711e42525a8/fhem/FHEM/HMConfig.pm#L519
we do not overwrite existing register definitions

File: examples/custom/contrib/FHEM/HMConfig_AskSinPPCustom.pm [hunk 2]
```diff
@@ -72,6 +72,15 @@ $HMConfig::culHmRegDefine{"longActNum"}                   = {a=>0xA5,s=>1.0,l=>3
...
+$HMConfig::culHmRegDefine{"transmitDevTryMax"} = {a=>20,s=>1,l=>0,min=>1,max=>10,c=>'',p=>'n',f=>'',u=>'',d=>1,t=>"transmitDevTryMax"};
```
Comment (pa-pa) [inline examples/custom/contrib/FHEM/HMConfig_AskSinPPCustom.pm:78]: can we reuse this https://github.com/mhop/fhem-mirror/blob/41fe9bbf317f54bccfc397e26224c711e42525a8/fhem/FHEM/HMConfig.pm#L526

**Review by pa-pa — COMMENTED** [review]
I'm happy to see new device.
Please change register as commented.

**Comment by DarkShortie** [comment]
Thanks for the hints, I've changed it like requested.

**Pull request merged**